pre_install_report/library/pre_install_check_permissions.py

```diff
@@ -42,6 +45,9 @@
 PROVISIONER_AZURE_DISK = "kubernetes.io/azure-disk"
 PROVISIONER_AWS_EBS = "kubernetes.io/aws-ebs"
 
+INGRESS_MIN_SUPPORTED_REL_EQ = '==1.18'
```

Comment:
I still think the name is confusing, once 1.19+ becomes available with v1 Ingress it won't be obvious how "MIN" version relates to using v1beta1

Reply:
Agreed.  I have modified the property name to INGRESS_V1BETA1_REL_EQ now.  Thanks